ons_alpha/bulletins/models.py

```diff
@@ -141,3 +157,8 @@ def previous_releases(self, request):
             context_overrides={"bulletins": previous},
             template="templates/pages/bulletins/previous_releases.html",
         )
+
+
+class BulletinTopicRelationship(Orderable):
```

Comment:
note on the convention: we tend to define the orderable above its related page model. something, something locality of behaviour and we've always done it.

not a blocker

Reply:
Good to know. Updated.